wrangler-service/pom.xml

```diff
@@ -29,7 +29,7 @@
     <app.main.class>io.cdap.wrangler.DataPrep</app.main.class>
     <tika.version>1.14</tika.version>
     <hadoop.version>2.3.0</hadoop.version>
-    <google.cloud.spanner.version>1.20.0</google.cloud.spanner.version>
+    <google.cloud.spanner.version>1.51.0</google.cloud.spanner.version>
```

Comment:
why do we need to change spanner version?

Reply:
To resolve dependency version conflicts, if we only update the bigquery version it causes the spanner connection to break due to a conflict in the protobuf library